Could not get unknown property 'android' for project ':app' of type org.gradle.api.Project

hello and thanks for the lib!

steps to reproduce

```
npx react-native init removeme --version 0.60.0
cd removeme
yarn add @notifee/react-native

echo "{
   "android": {
     "license": "XXXX.YYYY.ZZZZ"
   },
   "ios": {
     "license": "XXXX.YYYY.ZZZZ"
   }
 }" >> notifee.config.json

npx react-native run-android
```


output of build:

```
FAILURE: Build failed with an exception.

* Where:
Script '/Users/vojta/_dev/removemee/node_modules/@notifee/react-native/android/notifee-json.gradle' line: 253

* What went wrong:
A problem occurred configuring project ':@notifee_react-native'.
> Could not get unknown property 'android' for project ':app' of type org.gradle.api.Project.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 51s
```

since RN 60 is supported, can this be fixed? Thanks!

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -18,6 +18,8 @@ plugins {
   id "io.invertase.gradle.build" version "1.3"
 }
 
+project.evaluationDependsOn(':app')
+
 project.ext {
   set('react-native', [
     versions: [

